src/app/views/detailed_view.js

```diff
@@ -0,0 +1,43 @@
+import Backbone from 'backbone';
+import $ from 'jquery';
+import ContactView from 'app/views/detailed_view';
+import Contact from 'app/models/contact';
+
+const DetailedView = Backbone.View.extend({
+  initialize: function(options){
+    this.template = options.template;
+    // console.log("Inside DetailedView initialize");
+    // console.log(this.model);
+    // console.log(this.$el);
+    // this.on('click', this.closeIfOutOfFocus, this.$el.parent);
+  },
+  render: function(){
+    // this.delegateEvents();
+    // console.log("In detailed view render");
+    var result = this.template({name: this.model.get("name"), email: this.model.get("email"), phone: this.model.get("phone")});
+    var html = this.$el.html(result);
+    // console.log(result);
+    return this;
+  },
+  events: {
+    'click .edit-me': 'displayEditForm',
+    'click' : 'doNothing' //Catch a click on the form and stop it from bubbling up to the application_view
+  },
+  show: function(){
+    $('#contact-details').removeClass('hidden');
```

Comment:
You should use `this.$` to find elements within the view, or $el.removeClass('hidden') if `el` is the entire modal.  Using plain jQuery could cause the view to interact with something outside of it, which could cause unexpected behavior and have it interfere with other views.

It doesn't, here, but it could in a larger Backbone app.